docs/posts/annotations.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: default
+title: annotations
+description: Creating annotations
+---
+
+# Creating annotations in cellxgene
+
+We are _piloting_ a new feature in cellxgene that enables users to create and edit categorical annotations within the app.
```

Comment:
Perhaps tone should be 'you as the user are invited to pilot'?

Reply:
Great! Please revise.